other/edge-server/client/node-server/dist/js/db/connect.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
+    function adopt(value) { return value instanceof P ? value : new P(function (resolve) { resolve(value); }); }
+    return new (P || (P = Promise))(function (resolve, reject) {
+        function fulfilled(value) { try { step(generator.next(value)); } catch (e) { reject(e); } }
+        function rejected(value) { try { step(generator["throw"](value)); } catch (e) { reject(e); } }
+        function step(result) { result.done ? resolve(result.value) : adopt(result.value).then(fulfilled, rejected); }
+        step((generator = generator.apply(thisArg, _arguments || [])).next());
+    });
+};
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.getTodoCollection = exports.connectToEdgeServer = void 0;
+const mongodb_1 = require("mongodb");
+const uri = process.env.EDGE_SERVER_URI;
+// const uri = "mongodb://localhost:27021";
+console.log(uri);
```

Comment:
Should we remove these two lines from this file?

Reply:
Yep! Please remove.